android/app/src/main/java/com/chorestory/adapter/ParentQuestsAdapter.java

```diff
@@ -0,0 +1,39 @@
+package com.chorestory.adapter;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParentQuestsAdapter extends FragmentStatePagerAdapter {
+    private List<Fragment> fragmentList;
+    private List<String> fragmentTitleList;
+
+    public ParentQuestsAdapter(FragmentManager fragmentManager) {
```

Comment:
If we made the constructor accept 2 lists, and just set `this.fragmentList = fragmentList` for each then we can make the compiler do the work instead of `ParentQuestsFragment::onCreate` inserting list items every time it is called.

Reply:
@keller00 
@isabelleleeson 
I have made the same changes for `Parent Home` and `Child Home`

https://github.com/jimzhaozhengwang/CS446-Project/pull/28/commits/9984e5446c32930217ea192cc4479c71dcd54fcb